drivers/adc/adc_stm32.c

```diff
@@ -217,7 +217,7 @@ static void adc_stm32_enable_dma_support(ADC_TypeDef *adc)
 {
 	/* Allow ADC to create DMA request and set to one-shot mode as implemented in HAL drivers */
 
-#if defined(CONFIG_SOC_SERIES_STM32H7X)
+#if defined(CONFIG_SOC_SERIES_STM32H7X) || defined(CONFIG_SOC_SERIES_STM32H7RSX)
```

Comment:
Are you sure about that? I don't find the function LL_ADC_REG_SetDataTransferMode or LL_ADC_REG_SetDMATransferMode in the LL H7RS.
But there is the LL_ADC_REG_SetDMATransfer function, so I think that for H7RS we should fall into the `#else` case.

Reply:
correct, this is not valid for the ADC of the stm32h7RS